### PR TITLE
New db test

### PR DESCRIPTION
**Pull request by jbeorse**

### CODE REVIEW

**Comment by jbeorse** [comment]
Test comment to get Jenkins to build the PR

**Comment by wbrunette** [comment]
build

**Comment by jbeorse** [comment]
build

**Comment by wbrunette** [comment]
build

**Comment by wbrunette** [comment]
build

**Comment by wbrunette** [comment]
build

**Comment by wbrunette** [comment]
build

**Comment by wbrunette** [comment]
build

File: androidlibrary_lib/src/androidTest/java/org/opendatakit/utilities/OdkBaseTableTest.java [hunk 1]
```diff
@@ -169,4 +170,17 @@ public void testOdkTableParcelation() throws IOException {
     }
   }
 
+  @Test
```
Comment (nilesr) [inline androidlibrary_lib/src/androidTest/java/org/opendatakit/utilities/OdkBaseTableTest.java]: You can write `@Test (expected = IllegalArgumentException.class)` instead

Comment (jbeorse) [inline androidlibrary_lib/src/androidTest/java/org/opendatakit/utilities/OdkBaseTableTest.java]: Good idea. Updated

**Comment by jbeorse** [comment]
build

**Comment by wbrunette** [comment]
build

**Comment by jbeorse** [comment]
Test comment from Jenkins

**Comment by wbrunette** [comment]
build

**Pull request merged**